src/sicks3000.cc

```diff
@@ -123,8 +123,13 @@ int SickS3000::Close()
     // Read controller messages.
     if (serial->ReadPort(cReadBuffer, &read_bytes, 1999) < 0)
```

Comment:
It's not, but an output occurs in the inner method: https://github.com/clearpathrobotics/s3000_laser/blob/b065680230c0af5d9f04f350b5a7670ab2cc4e4b/external/SerialDevice/src/SerialDevice.cc#L434

I don't like this serial wrapper. I got partway through replacing it with [serial](https://github.com/wjwwood/serial), but I'd rather just rewrite the whole thing as BSD at some point, and then have proper nodelet/laser_pipeline support.